docs/source/debugger/cheats.rst

```diff
@@ -18,11 +18,11 @@ You can also type **help <command>** for further details on each command in the
 cheatinit
 ---------
 
-|  **cheatinit [<sign><width><swap>,[<address>,<length>[,<cpu>]]]**
+|  **cheatinit [<sign><width><swap>,[<address>,<length>[,<CPU>]]]**
```

Comment:
I'm not too sure about capitalising this in the placeholders.  In prose below, it should obviously be capitalised, but command line placeholders are a bit of a weird thing.

You also don't capitalise "bool" in command-line placeholders despite it being a proper noun, as an abbreviation of the surname "Boole".

There's also a case below where you haven't capitalised the acronym ASCII in a command-line placeholder, which is inconsistent (oh and ASCII is actually an acronym while CPU is an initialism or abbreviation, not an acronym).

Any other developers have an opinion?

Reply:
I did initially hesitate on capitalizing in the command structure section, but decided that making CPU look even across all instances was probably correct. If you decide otherwise, I'll be more than glad to change that back.

As for ASCII, I apparently missed that. I'll fix that. Consistency is important.